deployments/examples/ocis_individual_services/docker-compose.yml

```diff
@@ -212,13 +313,37 @@ services:
       OCS_IDM_ADDRESS: https://${OCIS_DOMAIN:-ocis.owncloud.test}
 
       OCS_JWT_SECRET: ${OCIS_JWT_SECRET:-Pive-Fumkiu4}
-      OCS_MACHINE_AUTH_API_KEY: ${OCIS_MACHINE_AUTH_API_KEY:-change-me-please}
+      OCS_MACHINE_AUTH_API_KEY: ${OCIS_MACHINE_AUTH_API_KEY:-changeme}
 
-      REVA_GATEWAY: storage-gateway:9142
+      REVA_GATEWAY: gateway:9142
     logging:
       driver: "local"
     restart: always
 
+  search:
+    image: owncloud/ocis:${OCIS_DOCKER_TAG:-latest}
+    deploy:
+      replicas: 1
+    networks:
+      ocis-net:
+    entrypoint:
+      - ocis
+      - search
+      - server
+    environment:
+      SEARCH_LOG_LEVEL: "${OCIS_LOG_LEVEL:-error}"
+      SEARCH_LOG_COLOR: "${OCIS_LOG_COLOR:-false}"
+      SEARCH_LOG_PRETTY: "${OCIS_LOG_PRETTY:-false}"
+
+      SEARCH_GRPC_ADDR: 0.0.0.0:9220
+
+      REVA_GATEWAY: gateway:9142
+      SEARCH_EVENTS_ENDPOINT: nats:9233
+
+      OCIS_MACHINE_AUTH_API_KEY: ${OCIS_MACHINE_AUTH_API_KEY:-changeme}
+      OCIS_SYSTEM_USER_API_KEY: ${SYSTEM_USER_API_KEY:-changme}
+      OCIS_SYSTEM_USER_ID: ${SYSTEM_USER_ID:-changeme}
+
```

Comment:
missing restart and log directive and a volume

Reply:
same as above (for the volume)